src/guessValidation.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+'use strict';
+
+const validateGuess = (guess) => {
+  const preparedGuess = guess
+    .split('')
+    .map(el => +el)
+    .sort((a, b) => a - b);
+
+  for (let i = 1; i < preparedGuess.length; i++) {
+    if (preparedGuess[i] === preparedGuess[i - 1]) {
+      console.log('Digits in your number must be unique');
```

Comment:
Will be better if you add a few more validations (now you check the uniqueness of the digits of the number, but check if you get a 4-digits number).